data/death_match_locations.json

```diff
@@ -120,6 +120,47 @@
             { "weapon_id" : 28, "ammo": 80085 }
         ],
     
+        "interior_id": 0,
+        "spawn_positions": [
+            { "position": [ -1471, 412, 32 ], "rotation": 0 },
+			{ "position": [ -1479, 413, 32 ], "rotation": 0 },
+			{ "position": [ -1404, 370, 32 ], "rotation": 0 },
+			{ "position": [ -1385, 375, 32 ], "rotation": 0 },
+			{ "position": [ -1428, 355, 32 ], "rotation": 0 },
+			{ "position": [ -1454, 369, 32 ], "rotation": 0 },
+			{ "position": [ -1405, 359, 32 ], "rotation": 0 },
+			{ "position": [ -1400, 386, 32 ], "rotation": 0 },
+			{ "position": [ -1467, 399, 32 ], "rotation": 0 },
+			{ "position": [ -1430, 394, 32 ], "rotation": 0 },
+			{ "position": [ -1431, 371, 32 ], "rotation": 0 },
+			{ "position": [ -1445, 388, 32 ], "rotation": 0 },
+			{ "position": [ -1429, 417, 32 ], "rotation": 0 },
+			{ "position": [ -1448, 345, 32 ], "rotation": 0 },
+			{ "position": [ -1453, 393, 32 ], "rotation": 0 },
+			{ "position": [ -1405, 413, 32 ], "rotation": 0 },
+			{ "position": [ -1443, 347, 32 ], "rotation": 0 },
+			{ "position": [ -1453, 365, 32 ], "rotation": 0 },
+			{ "position": [ -1453, 389, 32 ], "rotation": 0 }
```

Comment:
nit: tabs, spaces, something in between :p

Reply:
Probably used another editor for the JSON. Fixed :)